DataConnectors/Beyond Security beSECURE.json

```diff
@@ -0,0 +1,128 @@
+{
+  "id": "BeyondSecuritybeSECURE",
+  "title": "Beyond Security beSECURE",
+  "publisher": "Beyond Security",
+  "descriptionMarkdown": 
+  "The [Beyond Security beSECURE](https://beyondsecurity.com/) connector allows you to easily connect your Beyond Security beSECURE scan events, scan results and audit trail with Azure Sentinel, to view dashboards, create custom alerts, and improve investigation. This gives you more insight into your organization's network and improves your security operation capabilities.",
+  "graphQueries": [
+    {
+      "metricName": "Total data received",
+      "legend": "Beyond Security beSECURE",
+      "baseQuery": "union beSECURE_Audit_CL, beSECURE_ScanEvent_CL, beSECURE_ScanResults_CL"
+    }
+  ],
+  "sampleQueries": [
+	{
+	"description" : "CVSS Score Aggregated",
+	"query": "beSECURE_ScanResults_CL  | summarize count() by cvss_score_s"
+	},
+	{
+	  "description" : "Risk Aggregated",
+	  "query": "beSECURE_ScanResults_CL  | summarize count() by risk_name_s"
+	},
+	{
+	  "description": "Host Risk Aggregated",
+	  "query": "beSECURE_ScanResults_CL  | summarize count() by destination_host_s, risk_name_s"
+	},
+	{
+	  "description" : "Top 10 most vulnerable Hosts",
+	  "query": "beSECURE_ScanResults_CL  | \n where (risk_value_s == 8 or risk_value_s == 4) | \n summarize count() by destination_host_s, risk_value_s | \n order by risk_value_s, count_ | \n limit 10"
+	},
+  {
+    "description" : "Failed login",
+    "query": "beSECURE_Audit_CL | \bn where audit_event_s == 'Login failed' "

```

Comment:
Thanks, for adding additional queries - I validated and this query (pasted as follows) has a syntax error because of an ASCII char before 'n'. That's because of \bn - You can delete the b from \bn to make it \n and that should fix it. 
beSECURE_Audit_CL | n where audit_event_s == 'Login failed'